<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IN2056324W1" name="Test1" tests="3" failures="3" timestamp="5 Jul 2020 21:47:35 GMT" time="0.054" errors="0">
  <testcase name="Login" time="0.023" classname="com.automation.testNG.LoginTestParallelMethod">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.automation.testNG.LoginTestParallelMethod.Login(LoginTestParallelMethod.java:17)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- Login -->
  <testcase name="searchThanos" time="0.023" classname="com.automation.testNG.SearchProduct">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.automation.testNG.SearchProduct.searchThanos(SearchProduct.java:16)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- searchThanos -->
  <testcase name="searchThanos" time="0.002" classname="com.automation.testNG.SearchProduct">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.automation.testNG.SearchProduct.searchThanos(SearchProduct.java:16)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- searchThanos -->
</testsuite> <!-- Test1 -->
